backend/src/main/resources/application-demo.properties

```diff
@@ -0,0 +1,11 @@
+spring.mail.host=${NOODLE_MAIL_HOST}
+spring.mail.port=${NOODLE_MAIL_PORT}
+spring.mail.username=${NOODLE_MAIL_USER}
+spring.noodle.mail.from=${NOODLE_MAIL_FROM}
+spring.mail.password=${NOODLE_MAIL_PASSWORD}
+
+spring.noodle.security.expirationTime=${NOODLE_SECURITY_EXPIRATION_TIME}
+spring.noodle.security.secret=${NOODLE_SECURITY_SECRET}
+spring.noodle.security.hashPepper=pepper
```

Comment:
Isn't the pepper also a security secret which we might not want to have here? :o

Reply:
Although I guess it make sense insofar it's required for the demo data.